doc/glm/second_level_model.rst

```diff
@@ -0,0 +1,105 @@
+.. _second_level_model:
+
+===================
+Second level models
+===================
+
+.. topic:: **Page summary**
+
+   Second level models are used to take fMRI analysis to the group level. Once individual subject data has been processed in a common space (e.g. MNI, Talairach, or subject average), the data can be grouped and statistical tests  performed to make broader inferences on fMRI activity.
+
+.. contents:: **Contents**
+    :local:
+    :depth: 1
+
+
+Design matrix
+=============
+
+As with first level models, a design matrix needs to be defined before fitting a second level model. Some common second level models are one-sample (unpaired or paired) and two-sample t-tests. A quick look into setting up design matrices for these models is provided here.
+
+
+One-sample testing: unpaired
+----------------------------
+
+In one-sample testing, activity associated with a predictor variable is tested against some form of baseline activity. Therefore, predictors must be defined in order to set up a one-sample t-test. For example, consider the first level contrast of left-vs-right button press, which can be downloaded using :func:`nilearn.datasets.fetch_localizer_contrasts`::
+
+  from nilearn.datasets import fetch_localizer_contrasts
+  n_subjects = 16
+  data = fetch_localizer_contrasts(["left vs right button press"], n_subjects,
+                               get_tmaps=True)
+
+A first level model has been fit to this data and the contrasts maps for left-vs-right button press activation are available in the `cmaps` parameter. The second level model is fit using these contrast maps thus::
+
+  from nilearn.stats.second_level_model import make_second_level_design_matrix
+  second_level_input = data['cmaps']
+  design_matrix = pd.DataFrame([1] * len(second_level_input), columns=['intercept'])
```

Comment:
Here also, there shouldn't be code snippets. Just pointers to functions, classes, and examples.